packages/server/graphql/intranetSchema/sdl/resolvers.ts

```diff
@@ -0,0 +1,26 @@
+/* This file dynamically requires all queries, mutations, and types.
+ * No need to mess with this unless we add subscriptions to the private schema
+ */
+import path from 'path'
+import {Resolvers} from './resolverTypes'
+
+const importAll = (context: __WebpackModuleApi.RequireContext) => {
+  const collector = {} as Record<string, any>
+  context.keys().forEach((relativePath) => {
+    const {name} = path.parse(relativePath)
+    collector[name] = context(relativePath).default
+  })
+  return collector
+}
+
+const resolverMap: Resolvers = {
```

Comment:
@blimmer this is my attempt to emulate what you were doing with that codegen script

Reply:
my current thinking is to have 3 folders inside `/graphql`:
- public
  - public/queries
  - public/mutations
  - public/subscriptions
  - public/types
- private
  - private/queries
  - private/mutations
  - private/types
- github
  - queries
  - mutations
- gitlab
  - queries
  - mutations

the above won't touch the current hierarchy so we can gradually move over to this new format.